src/Console/Application.php

```diff
@@ -63,6 +69,21 @@ public function __construct()
         ));
     }
 
+    public function enableDeprecationWarnings()
+    {
+        $this->deprecationWarningsEnabled = true;
+    }
+
+    /**
+     * @param string $message
+     */
+    public static function triggerDeprecation($message)
```

Comment:
i don't see value of this proxy method... ?

Reply:
Only deprecations triggered by calling this method will be printed.

Reply:
why not all of them ?

Reply:
This is an edge case but one may use custom fixers requiring vendor code that triggers deprecation notices unrelevant to CLI usage.

Reply:
Also, some deprecation notices are not intended for CLI users but for package authors, e.g. deprecated classes or methods. I don't think we should show a warning for those notices.